utils/oscap-docker.in

```diff
@@ -55,7 +55,10 @@ class OscapDocker(object):
 
 def ping_docker():
     ''' Simple check if the docker daemon is running '''
-    client = docker.Client()
+    try:
+        client = docker.Client()
+    except AttributeError:
```

Comment:
try the new class name first, fallback to old deprecated class, comment why it's done